fluidsynth/src/sfloader/fluid_defsfont.h

```diff
@@ -435,7 +435,7 @@ struct _fluid_defpreset_t
 {
   fluid_defpreset_t* next;
   fluid_defsfont_t* sfont;                  /* the soundfont this preset belongs to */
-  char name[21];                        /* the name of the preset */
+  char name[22];                        /* the name of the preset */
```

Comment:
> I mean, if the incoming SF2 preset name is empty, why tinker with it and not simply leave it empty?

I dont know why it was done like this, but I'm not feeling comfortable breaking with this convention either. I can also understand your point, it IS a very rare case. So taking all of that into account again we might be best doing to change the default string as you suggested. Feel free to create another PR.